plugin/evm/atomic/txpool/mempool.go

```diff
@@ -575,7 +576,13 @@ func (m *Mempool) RemoveTx(tx *Tx) {
 // addPending makes sure that an item is in the Pending channel.
 func (m *Mempool) addPending() {
 	select {
-	case m.Pending <- struct{}{}:
+	case m.pending <- struct{}{}:
 	default:
 	}
 }
+
+// SubscribePendingTxs implements the BuilderMempool interface and returns a channel
+// that signals when there is at least one pending transaction in the mempool
+func (m *Mempool) SubscribePendingTxs() <-chan struct{} {
+	return m.pending
+}
```

Comment:
This will be required in upcoming PRs (it's also nice that we can unexport the channel, hope it's safe)